### DISCUSSION TITLE
content folder location

### DISCUSSION THREAD

**Discussion by heat5** [Difegue/LANraragi#708] [Q&A] [answered]
How to create a second content folder location in windwos LR  ?  2 hard drives

**Comment by Difegue** [comment, marked as answer]
👋 I don't offer functionality to have multiple content folders, but you can aggregate multiple folders in one virtual location using [symlinks.](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/)

**Reply by heat5** [reply]
thank you
